frontend/package.json

```diff
@@ -32,6 +32,7 @@
     "httpbis-digest-headers": "^1.0.0",
     "isbot": "^5.1.32",
     "lit": "^3.3.1",
+    "lodash": "^4.17.21",
```

Comment:
Let's not add lodash, as we can get carried away with using it too much when it's here. Maybe https://www.npmjs.com/package/fast-equals would be better alternative - should check impact on bundle size first.